app/controllers/v0/beta_registrations_controller.rb

```diff
@@ -12,5 +12,10 @@ def create
       BetaRegistration.find_or_create_by(user_uuid: current_user.uuid, feature: params[:feature])
       render json: { 'user': current_user.email, 'status': 'OK' }
     end
+
+    def destroy
+      BetaRegistration.where(user_uuid: current_user.uuid, feature: params[:feature]).first.try(:destroy)
```

Comment:
Total nit, but if you'd like you could do:

```ruby
BetaRegistration.find_by(user_uuid: current_user.uuid, feature: params[:feature])&.destroy
```

Reply:
Nice use of safe navigation operator; I'll put this in